crates/router/src/connector/stripe/transformers.rs

```diff
@@ -2956,6 +2969,8 @@ pub struct WebhookEventObjectData {
     pub created: PrimitiveDateTime,
     pub evidence_details: Option<EvidenceDetails>,
     pub status: Option<WebhookEventStatus>,
+    #[serde(flatten)]
+    pub metadata: Option<ConnectorMetaData>,
```

Comment:
is this struct working for web hooks? we have serde rename for the fields in the struct, but connector is giving us proper json